flask_api/.gitignore

```diff
@@ -1,2 +1,3 @@
 venv/*
 __pycache__/*
+env0/*
```

Comment:
good catch, you can replace line 1 with line 3. `venv` here was an arbitrary name for the virtual env, but we're calling it `env0` so line 1 doesn't do anything

Reply:
I believe you've made the changes so I'll close this pull request